### PR TITLE
Updated deprecated functions

### PR DESCRIPTION
**Pull request by andrewkharook**
Updated deprecated functions
Minor bugfixes

### CODE REVIEW

File: ddtf.js [hunk 1]
```diff
@@ -87,30 +85,29 @@ $.fn.ddTableFilter = function(options) {
...
       console.log('Build: ' + (stop.getTime() - start.getTime()) + 'ms');
     }
   });
-  
-}
+};
 
 $.fn.ddTableFilter.defaultOptions = {
   valueCallback:function() {
-    return escape($.trim($(this).text()));
```
Comment (andrewkharook) [inline ddtf.js]: `escape` function is deprecated, using `encodeURIComponent()` instead.

File: ddtf.js [hunk 2]
```diff
@@ -87,30 +85,29 @@ $.fn.ddTableFilter = function(options) {
...
     }
-    
+
     if(options.debug) {
       var stop = new Date();
       console.log('Build: ' + (stop.getTime() - start.getTime()) + 'ms');
     }
   });
-  
-}
```
Comment (andrewkharook) [inline ddtf.js]: Unterminated line

File: ddtf.js [hunk 3]
```diff
@@ -62,16 +62,14 @@ $.fn.ddTableFilter = function(options) {
           console.log('Search: ' + (changeStop.getTime() - changeStart.getTime()) + 'ms');
         }
         refreshFilters(table);
-        var changeStop = new Date();
```
Comment (andrewkharook) [inline ddtf.js]: Duplicate variable `changeStop`

**Comment by rbayliss** [comment]
Looks great.  Thanks!

**Pull request merged**